sugarscape.py

```diff
@@ -997,6 +1033,8 @@ def verifyConfiguration(configuration):
 
     # Set timesteps to (seemingly) unlimited runtime
     if configuration["timesteps"] < 0:
+        if "all" in configuration["debugMode"] or "agent" in configuration["debugMode"]:
+            print("Cannot have a negative amount of timesteps. Setting timesetup to unlimited runtime.")
```

Comment:
Maybe the second sentence could just be "setting runtime to unlimited" for clarity? Alternatively, does it make sense to tell the user they "cannot have" a negative amount of timesteps when we already specified that `-1` means infinite timesteps?